bus-mapping/src/circuit_input_builder.rs

```diff
@@ -1679,6 +1679,13 @@ pub fn gen_state_access_trace<TX>(
         let i = Some(index);
         let (contract_address, code_source) = &call_stack[call_stack.len() - 1];
         let (contract_address, code_source) = (*contract_address, *code_source);
+
+        let (mut push_stack, mut pop_stack) = (false, false);
```

Comment:
In the EVM context there's many places where push_stack and pop_stack could be understood as pushing a word to the stack or popping a word to the stack.  I suggest renaming these variables to `push_call_stack` and `pop_call_stack` to make the meaning more clear.

Reply:
The variables had been renamed.